README.md

```diff
@@ -903,28 +903,30 @@ Translations of the guide are available in the following languages:
   ```
 
 * <a name="no-and-or-or"></a>
-  The `and` and `or` keywords are banned. It's just not worth it. Always use
-  `&&` and `||` instead.
+  The `and` and `or` keywords are banned. The minimal added readability is just
+  not worth the high probability of introducing subtle bugs. For boolean
+  expressions, always use `&&` and `||` instead. For flow control, use
+  `if` and `unless`; `&&` and `||` are also acceptable but less clear.
 <sup>[[link](#no-and-or-or)]</sup>
 
   ```Ruby
   # bad
   # boolean expression
-  if some_condition and some_other_condition
```

Comment:
Because they demonstrate the subtle bugs that `and`/`or` can introduce.  In the bad one (and the other bad one I just chopped out), the variable is set to the opposite of the value that it would have been had the author used `&&`/`||`.